sdk/identity/identity/README.md

```diff
@@ -8,9 +8,34 @@ to authenticate API requests. It supports token authentication using an Azure Ac
 
 ### Prerequisites
 
-- an Azure subscription
-  - if you don't have one, you can sign up for a [free account](https://azure.microsoft.com/free/)
 - Node.js 8 LTS or higher
+- An Azure subscription.
+    - You can sign up for a [free account](https://azure.microsoft.com/free/).
+- The [Azure CLI][azure_cli] can also be useful for authenticating in a development environment, creating accounts, and managing account roles.
+
+#### Authenticating via Visual Studio Code
+
+Developers using Visual Studio Code can use the [Azure Account Extension](https://marketplace.visualstudio.com/items?itemName=ms-vscode.azure-account), to authenticate via the IDE. Applications using the `DefaultAzureCredential` or the `VisualStudioCodeCredential` can then use this account to authenticate calls in their application when running locally.
+
+To authenticate in Visual Studio Code, first ensure the [Azure Account Extension](https://marketplace.visualstudio.com/items?itemName=ms-vscode.azure-account) is installed. Once the extension is installed, press `F1` to open the command palette and run the `Azure: Sign In` command.
+
+![Visual Studio Code Account Sign In](./images/VsCodeLoginCommand.png)
+
+#### Authenticating via the Azure CLI
+
+Developers coding outside of an IDE can also use the [Azure CLI][azure_cli] to authenticate. Applications using the `DefaultAzureCredential` or the `AzureCliCredential` can then use this account to authenticate calls in their application when running locally.
```

Comment:
Coding inside an IDE and using the Azure CLI are not incompatible. We should maybe just say "Developers using the `[Azure CLI]` tool can use it to authenticate.